ui/src/dashboards/components/variablesControlBar/VariablesControlBar.tsx

```diff
@@ -163,7 +164,12 @@ const mstp = (state: AppState): StateProps => {
   } = state
 
   return {
-    variables,
+    variables: filterUnusedVars(
+      variables,
+      Object.values(state.resources.views.byID).filter(
+        v => v.dashboardID === dashboardID
```

Comment:
```suggestion
        variable => variable.dashboardID === dashboardID
```
just to improve searchability

Reply:
👍